.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Release Check List
+about: v<TARGET_RELEASE>
+title: "v<TARGET_RELEASE Check List"
+---
+
+## v<TARGET_RELEASE>
+
+* [ ] - Get the latest ocicrypt-rs used as part of image-rs
+* [ ] - Get the image-rs, using the latest ocicrypt-rs, as part of attestastion-agent
+- [ ] - Get the latest attestation-agent, using the latest image-rs, as part of the Enclave CC
+- [ ] - Get the latest attestation-agent, using the latest image-rs, as part of the Kata Containers
+- [ ] - Get the latest td-shim as part of the Kata Containers
+- [ ] - Get the latest Enclave CC payload tested as part of the Operator
+- [ ] - Get the latest Kata Containers payload tested as part of the Operator
+- [ ] - Cut an ocicrypt-rs v<TARGET_RELEASE> release
+- [ ] - Cut an image-rs v<TARGET_RELEASE> release, using ocicrypt-rs v<TARGET_RELEASE> release
+- [ ] - Cut an attestation-agent v<TARGET_RELEASE> release, using the image-rs v<TARGET_RELEASE> release
+- [ ] - Cut a td-shim v<TARGET_RELEASE> release
```

Comment:
Do we need an extra step here to cover updating kata-containers to use the tagged release of the attestation-agent and image-rs once they've been cut, but before we've tagged the Kata containers